sdk/src/driftClient.ts

```diff
@@ -6504,7 +6549,11 @@ export class DriftClient {
 			  }
 			: undefined;
 
-		if (tx instanceof VersionedTransaction) {
+		const version = (tx as VersionedTransaction)?.version;
+		const isVersionedTx = ((tx instanceof VersionedTransaction || version !== undefined));
```

Comment:
this is a funny one too .. before this change I was actually catching v0 Versioned Transactions that were failing the check and being sent through the `txSender.send` flow

Reply:
should note I confirmed that sending these txs still works when I disabled versioned transactions from UI settings .. a VersionedTransactions with version "legacy" is fine to go through the "sendVersionedTransaction" flow